### DISCUSSION TITLE
What is the font used for System Default?

### DISCUSSION THREAD

**Discussion by maximillianus** [x1unix/go-playground#344] [Q&A] [answered]
Hi I wonder what is the System Default font used? I am pretty sure it is not `Cascadia Code` as mentioned in the website since I have compared it with actual `Cascadia Code`. 

I find the System default font nicer than Cascadia Code. I am guessing it is `Menlo` but the spacing and weight looks lighter. Kindly let me know what it is, it will greatly relieve my eyes 🙂.

**Comment by x1unix** [comment, marked as answer]
Hi.

Here are default font settings - https://github.com/x1unix/go-playground/blob/master/web/src/services/fonts.ts#L12

```css
font-family: Menlo, Monaco, Consolas, "Lucida Console", "Roboto Mono", "Courier New", monospace
```

Please also note that font rendering depends on OS, browser and your display.
Browsers render fonts differently and font quality depends on your display (orientation, type, etc).

**Reply by maximillianus** [reply]
Much appreciated! Thank you for writing this golang tools! 😃
